lib/bsdlib/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 zephyr_library()
+zephyr_library_sources(bsdlib.c)
```

Comment:
I wonder about `bsd_sanity.c` - now as we have a dedicated `bsdlib.c` source file, could we move `bsd_sanity.c` contents into it? All in all, it's just a check, I don't think it requires a separate source file.

Reply:
Moved inside `bsdlib.c`